docker-compose.yml

```diff
@@ -21,4 +21,48 @@ services:
         ports:
             - 8080:8080
         depends_on:
-            - redis
\ No newline at end of file
+            - redis
+
+    redis2:
+        image: redis:latest
+        container_name: redis2
+        ports:
+            - 6380:6380
+        volumes:
+            - ./config/redis.conf:/redis.conf
+        command: [ "redis-server", "/redis.conf", "--port", "6380" ]
+
+    gmm_server2:
+        build: .
+        container_name: gmm_server2
+        stdin_open: true
+        environment:
+            - ZONE_ID=zone2
+            - GMM_REDIS_HOST=redis2
+            - GMM_REDIS_PORT=6380
+        ports:
+            - 8081:8080
+        depends_on:
+            - redis2
+
+    redis3:
+        image: redis:latest
+        container_name: redis3
+        ports:
+            - 6381:6381
+        volumes:
+            - ./config/redis.conf:/redis.conf
+        command: [ "redis-server", "/redis.conf", "--port", "6381" ]
```

Comment:
Aha, czyli po prostu każda zona ma wpisany na twardo port redisa. No ok, na razie niech będzie

Reply:
z tego co rozmawiałem z Adrianem, to tak teraz to robimy (trochę na chama)

Reply:
według tego, co mówił Łukasz, multiplikacją bardziej mamy się przejmować gdy będziemy to wynosić na kubernetesa